src/components/Toast/Toast.styled.ts

```diff
@@ -21,7 +24,13 @@ export const Container = styled.div<ToastContainerProps>`
   ${({ placement }) => getPlacement(placement)}
 `
 
-export const Element = styled.div<Pick<ToastElementProps, 'transform' | 'transitionDuration' | 'placement'>>`
+interface StyledToastProps extends Pick<ToastElementProps,
+| 'transform'
+| 'transitionDuration'
+| 'placement'
+| 'zIndex'>{}
```

Comment:
diff를 위해서 마지막 줄의 꺽쇠 이후부터 다른 라인에 적는게 좋지 않을까 싶습니다.
```suggestion
| 'zIndex'
>{}
```